tests/system/test_system.py

```diff
@@ -460,11 +462,10 @@ def test_bucket_get_blob_with_user_project(self):
 
 class TestStorageFiles(unittest.TestCase):
 
-    DIRNAME = os.path.realpath(os.path.dirname(__file__))
     FILES = {
-        "logo": {"path": DIRNAME + "/data/CloudPlatform_128px_Retina.png"},
-        "big": {"path": DIRNAME + "/data/five-point-one-mb-file.zip"},
-        "simple": {"path": DIRNAME + "/data/simple.txt"},
+        "logo": {"path": DATA_DIRNAME + "/CloudPlatform_128px_Retina.png"},
+        "big": {"path": DATA_DIRNAME + "/five-point-one-mb-file.zip"},
+        "simple": {"path": DATA_DIRNAME + "/simple.txt"},
```

Comment:
Won't that be an error on Windows that use a different path separator?

(the change itself is good, referring to the existing state here)